src/utils/cookie.ts

```diff
@@ -22,6 +22,12 @@ export const COOKIE_KEYS = {
 export const hasAcceptedCookies = () =>
   Cookies.get(COOKIE_KEYS.ACCEPT_COOKIES_KEY) === 'true';
 
+/**
+ * @returns {boolean} whether the user is authenticated
+ */
+export const isUserAuthenticated = () =>
+  Boolean(Cookies.get(COOKIE_KEYS.SESSION_KEY));
```

Comment:
We should probably not rely on this since we should set the session cookie as httpOnly

Reply:
Indeed, but I just factor it out in sdk, so we can rely on the function but change it overall to rely on another cookie "isAuthenticated" for example. This function is actually useful to quickly fallback on public endpoints, avoiding triggering private requests if we 're anyway logged out.